python/test/ngraph/test_ops_fused.py

```diff
@@ -100,3 +100,26 @@ def test_clamp_operator_with_array():
     result = computation()
     expected = np.clip(data_value, min_value, max_value)
     assert np.allclose(result, expected)
+
+
+def test_grn_operator():
+    runtime = get_runtime()
+
+    data_value = np.arange(start=1.0, stop=25.0, dtype=np.float32).reshape(1, 2, 3, 4)
```

Comment:
`np.arange` 

Do you mean "arrange", like the arrangement of the reshape?  Or do you mean "a range" between values?

Reply:
I mean "a range", array of evenly spaced values. `np.arange` returns `ndarray`.

Reply:
https://docs.scipy.org/doc/numpy/reference/generated/numpy.arange.html